src/utils.rs

```diff
@@ -66,3 +66,12 @@ macro_rules! r_expect {
         }
     }};
 }
+
+#[macro_export]
+macro_rules! r_vec {
+    ($($x:tt)*) => {
+        {
+            Obj::Vector(Vector::from(vec![$($x)*]))
+        }
+    }
+ }
```

Comment:
Instead of giving this a custom macro, I think implementing `From` would be just about as succinct, but without any macro magic.

I already feel like the project leans a bit too hard on macros so I'm a bit sensitive to introducing new ones right now.

Reply:
Also fine for me! Thanks for the quick and great feedback! :)